frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -280,6 +343,17 @@ export default function CourseDescription({ structure }) {
                   </Tag>
                 );
               })}
+            <Title level={3} className="text cs-final-attr">
+              Capacity
+            </Title>
+            {courseCapacity.enrolments ? (
+                <ProgressBar
+                  progress={Math.round((courseCapacity.enrolments/courseCapacity.capacity) * 100)}
+                  height = {30}
+                />
+            ) : (
+              <Text>Nothing to show</Text>
```

Comment:
Is it supposed to have nothing to show at the moment?
Is this because registrations are not open? If so, during what times will this feature typically show a number.
<img width="421" alt="image" src="https://user-images.githubusercontent.com/93496985/163664990-b60806e5-bcc6-4eab-9734-e3951e822229.png">
For context, that's COMP2521 but this result appears for all courses.